src/app/utils/utils.ts

```diff
@@ -1,6 +1,8 @@
 export function accessibleRouteChangeHandler() {
-  const mainContainer = document.getElementById('primary-app-container');
-  if (mainContainer) {
-    mainContainer.focus();
-  }
+  setTimeout(() => {
```

Comment:
Isn't there some promise we can listen to for `onRouteChange` or something similar?

Reply:
Yes, we can listen to route changes, but that's twice removed from when we need this code to actually run. The problem is that the route change event fires immediately on selecting a route to navigate to. This event fires before even synchronous components have mounted.

Waiting on a component to fully mount also isn't enough, as their needs to exist a brief delay so that assistive technology has time to notice the new content. This ensures that screen readers do whatever the default behavior is when encountering new content (usually reading it aloud).

I don't love using a timeout here, but it does seem to behave as expected from an accessibility standpoint. Feel free to recommend a different pattern, I'm happy to use whatever the best technique is as long as the overall strategy remains (having new content announced after route changes).

Does it make sense? Let me know if you have any thoughts on how to better treat this.

Reply:
> their needs to exist a brief delay so that assistive technology has time to notice the new content.

This makes sense after explaining how they are constantly diffing the DOM. We should use `(JAW's polling rate) + 1` as the value here.

Reply:
Agree. I've actually been unable to find an exact number on what the polling rate is. If you happen to know, I'm happy to use that. In the meantime, I've used a default of 50ms which had success in the past. We can always update it to be more accurate as we have time to do further research. The thing that's certain right now is that without a delay it doesn't work in JAWS. 50ms isn't really a perceivable difference so I think it's a good place to start.